server/webapp/WEB-INF/rails/webpack/views/components/forms/input_fields.tsx

```diff
@@ -572,6 +572,66 @@ export class TriStateCheckboxField extends FormField<TriStateCheckbox> {
   }
 }
 
+export interface RadioData {
```

Comment:
Do we need to export this?

Reply:
Hmm, apparently we don't need to export it. I thought it was needed since we're passing this object from the page where the radio buttons will be used.

Reply:
That interface is purely for type-checking, not exporting will still allow typechecking, auto-complete hints etc.

Only reason to export might be if you want to use that "named interface" somewhere, (`foo extends ...`, define a variable )